staging/src/k8s.io/apiserver/pkg/admission/interfaces.go

```diff
@@ -63,8 +63,8 @@ type Interface interface {
 
 // ValidationInterface is an abstract, pluggable interface for Admission Control decisions.
 type ValidationInterface interface {
-	// Validate makes an admission decision based on the request attributes.  It is NOT allowed to mutate
-	Validate(a Attributes) (err error)
+	// ValidatingAdmit makes an admission decision based on the request attributes.  It is NOT allowed to mutate
+	ValidatingAdmit(a Attributes) (err error)
```

Comment:
What about just `Validate` as a name?

Reply:
taken already by the Validator interface below. We might rename that to ValidatorInitialization if we want and use Validate here.

Reply:
have renamed Validate to ValidateInitialization and ValidatingAdmit to Validate.